src/Symfony/Component/Security/Http/LoginLink/LoginLinkHandlerInterface.php

```diff
@@ -24,7 +24,7 @@ interface LoginLinkHandlerInterface
     /**
      * Generate a link that can be used to authenticate as the given user.
      */
-    public function createLoginLink(UserInterface $user, Request $request = null): LoginLinkDetails;
+    public function createLoginLink(UserInterface $user, Request $request = null /*, int $lifetime = null */): LoginLinkDetails;
```

Comment:
we need an `@param` annotation (so that userland implementations are noticed by DebugClassLoader IIRC)

Reply:
Indeed, added now.